core/net/mac/csma.c

```diff
@@ -164,7 +164,7 @@ transmit_packet_list(void *ptr)
 }
 /*---------------------------------------------------------------------------*/
 static void
-free_packet(struct neighbor_queue *n, struct rdc_buf_list *p)
+csma_free_packet(struct neighbor_queue *n, struct rdc_buf_list *p)
```

Comment:
as per Contiki code style, static functions should not be prepended by the module name